src/jdk.jdi/share/classes/com/sun/jdi/connect/spi/Connection.java

```diff
@@ -102,6 +102,7 @@ public Connection() {}
      * @throws  java.io.IOException
      *          If the length of the packet (as indictaed by the first
      *          4 bytes) is less than 11 bytes, or an I/O error occurs.
+     * @spec jdwp/jdwp-spec.html Java Debug Wire Protocol
```

Comment:
http://cr.openjdk.java.net/~jjg/8296546/api.00/jdk.jdi/com/sun/jdi/connect/spi/Connection.html#readPacket()

Within this javadoc page the jdwp-spec.html references are titled "JDWP Specification", but these `@spec` references are titled "Java Debug Wire Protocol". I suggest making them more consistent. There is one more case below and this same issue also applies to TransportService.java. Perhaps the title in jdwp-spec.html should be updated. I think "Java Debug Wire Protocol (JDWP) Specification" would be good.